proposals/low-level-struct-improvements.md

```diff
@@ -727,9 +727,14 @@ For a `new` expression with initializers, the initializer expressions count as a
 
 ## Changes in unsafe context
 
-Pointer types are extended to allow managed types as referent type, but only in casts.
+Pointer types (section 22.3) are extended to allow managed types as referent type.
```

Comment:
>section 22.3 [](http://example.com/codeflow?start=15&length=12)

Consider adding links to [https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/unsafe-code.md](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/unsafe-code.md) sections.